chain/store.go

```diff
@@ -52,6 +53,19 @@ const (
 	PostgreSQL StorageType = "postgres"
 )
 
+func MetricsStorageType(st StorageType) int {
+	// NOTE: Please only append to this list.
```

Comment:
is there a reason not to use some consts/iotas for this?

Reply:
The `StorageType` is a string as that's what comes over the CLI switches.
So, we'd need a second type to use `iota`. As for constants, we'd make the edit in a separate place again.
Finally, it's the only place we use this. If there would be at least another usage place, I'd definitely use a const/iota style for this, as suggested.
That being said, I don't have a strong preference for this, so I can switch it to a iota style value definition.